_source/handbook/03_page_refreshes.md

```diff
@@ -0,0 +1,82 @@
+---
+permalink: /handbook/page_refreshes.html
+description: "Turbo can perform smooth page refreshes with morphing and scroll preservation."
+---
+
+# Smooth page refreshes with morphing
+
+[Turbo Drive](/handbook/drive.html) makes navigation faster by avoiding full-page reloads. But there is a scenario where Turbo can raise the fidelity bar further: loading the current page again (page refresh).
+
+A typical scenario for page refreshes is submitting a form and getting redirected back. In such scenarios, sensations significantly improve if only the changed contents get updated instead of replacing the `<body>` of the page. Turbo can do this on your behalf with morphing and scroll preservation.
+
+${toc}
+
+## Morphing
```

Comment:
Is it worth explaining the event lifecycle? Is it worth linking to the [events themselves](https://github.com/hotwired/turbo-site/blob/f1a52cbdd462981a4aa99f04017a1d957b8a5534/_source/reference/events.md#L35-L41)?


For what it's worth, https://github.com/hotwired/turbo-site/pull/145 introduces linkable headings for each event (in addition to tables of `event.details` type information).

For example, we'd be able to link directly to the [`turbo:morph`](https://github.com/hotwired/turbo-site/pull/145/files#diff-800ab9d00bce4ff48bd1ce8ca1ed47c9d6aae5ab03a280b8fa29cb9e8dacbefaR107) event.